Firmware/menu.cpp

```diff
@@ -233,7 +215,9 @@ static void menu_draw_item_select_sheet_E(char type_char, const Sheet &sheet)
     lcd_set_cursor(0, menu_row);
     SheetFormatBuffer buffer;
     menu_format_sheet_select_E(sheet, buffer);
-    lcd_printf_P(PSTR("%c%-18.18s%c"), menu_selection_mark(), buffer.c, type_char);
+    lcd_putc(menu_selection_mark());
+    lcd_print_pad_P(buffer.c, LCD_WIDTH - 2);
```

Comment:
buffer.c is in ram. Use the non-P version of lcd_print_pad

Reply:
Fixed [8207561](https://github.com/prusa3d/Prusa-Firmware/pull/3873/commits/82075610040c6fe4ea44ea7834bc7a7027f34e8d)